docs/source/multiprocessing.rst

```diff
@@ -1,9 +0,0 @@
-Parallel Execution
-==================
-
-The tools package includes utilities that makes it easy to run a set of requests in parallel using a multiprocessing pool:
-
-    >>> notes = list(tools.iterget_notes(client, invitation='ICLR.cc/2019/Conference/-/Blind_Submission'))
-    >>> openreview.tools.parallel_exec(notes, do_work)
```

Comment:
we could keep this file but change the documentation and mention that `tools.concurrent_get` should do the trick

Reply:
cc @carlosmondra

Reply:
There is no section in the docs for this. If we were to add it, it should go under the Reference section. I would suggest having an openreview-py subsection under Reference. We could also link readthedocs from this subsection.